components/inputs/input-date.js

```diff
@@ -376,7 +376,10 @@ class InputDate extends SkeletonMixin(FormElementMixin(LocalizeCoreElement(LitEl
 	}
 
 	_handleDropdownOpen() {
-		if (this._dropdown && !this._dropdown.openedAbove) this.shadowRoot.querySelector('d2l-focus-trap').scrollIntoView({ block: 'nearest', behavior: 'smooth', inline: 'nearest' });
+		if (this._dropdown && !this._dropdown.openedAbove) {
+			this._dropdown.shadowRoot.querySelector('d2l-focus-trap')
+				.scrollIntoView({ block: 'nearest', behavior: 'smooth', inline: 'nearest' });
```

Comment:
Hmm, the reaching into dropdown's shadow root to do the scrolling is a little unfortunate. Do we know what this was being done for initially? Wondering if we still need it now with the mobile tray...

Reply:
Did some digging here. It's used in the case of short screens (not necessarily mobile) where the dropdown gets scrolled into view. Looks like replacing focus-trap with calendar works as well, since they are the same size.